platform/broadcom/sonic-platform-modules-dell/s6000/scripts/override.conf

```diff
@@ -0,0 +1,3 @@
+[Service]
+ExecStart=
```

Comment:
Any intention for this empty line?

Reply:
This is just to clear the existing options if any. Followed the same procedure in rest of the platforms as well. https://github.com/Azure/sonic-buildimage/blob/master/platform/broadcom/sonic-platform-modules-dell/s6100/scripts/override.conf